.env.test

```diff
@@ -3,10 +3,6 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 FLASK_SKIP_DOTENV=True
 
-# Change the following values for local development.
-SECRET_KEY=secret-key
 
-# Updated these variables to reflect your own Trello environment
-TRELLO_AUTH='testauth1234'
-TRELLO_TOKEN='testtoken1234'
-TRELLO_BOARD_ID='abcd1234'
\ No newline at end of file
+MONGO_DB_CONNECTION=mongodb://fakemongo.com
+MONGO_DB_NAME=test_todo_app
```

Comment:
It would be nice to also add placeholder values for these fields to your `.env.template` file